test/rendering-test/test-config.js

```diff
@@ -17,8 +17,8 @@ import {
 
 } from 'deck.gl';
 
-export const WIDTH = 1600;
-export const HEIGHT = 900;
+export const WIDTH = 800;
```

Comment:
There is some downscaling support in luma.gl if we want to render in high resolution but show smaller images